File: Mar_PHP/Assignments/BasicPrograms/25.php
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>LCM Calculator</title>
</head>
<body>
    <h2>Calculate LCM of Two Numbers</h2>
    <form method="post">
        Enter first number: <input type="number" name="num1" required>
        <br><br>
        Enter second number: <input type="number" name="num2" required>
        <br><br>
        <input type="submit" name="submit" value="Calculate LCM">
    </form>

    <?php
    function gcd($a, $b) {
        return $b ? gcd($b, $a % $b) : $a;
    }

    function lcm($a, $b) {
        return abs($a * $b) / gcd($a, $b);
    }

    if (isset($_POST['submit'])) {
        $num1 = $_POST['num1'];
        $num2 = $_POST['num2'];
        
        
        $lcm = lcm($num1, $num2);

        echo "<p>LCM of $num1 and $num2 is: $lcm</p>";
    }
    ?>
</body>
</html>

File: Jan_Basic_Programming/Assignments/C/25.c
#include<stdio.h>
// LCM of 2 Nums

int main(){
    int n1, n2, cn, lcm;

    printf("Enter 2 Numbers : ");
    scanf("%d %d", &n1, &n2);

    if(n1>n2){
        cn=n1;
    }else{
        cn=n2;
    }

    while(1){
        if(cn % n1 == 0 && cn % n2 ==0){
            lcm = cn;
            break;
        }
        cn++;
    }

    printf("lcm(%d, %d) : %d", n1, n2, cn);
 return 0;
}